### ISSUE TITLE
Suggestion: Decouple framework specific aspects.

### ISSUE THREAD

**Issue by rubybb** [praveenjuge/kutty#43]
Kutty seems to be created with Alpine.js, and while it's a great library, not everyone is using it and it's just dead code if they import Kutty into their project solely for the TailwindCSS configuration just to implement their own components in React, Vue, or whatnot.

And so in my opinion; Alpine.js should not be a *requirement* for this project by default, and that logic should be seperated into their own areas, and perhaps even implement React, vue, ect variants.

Let me know on your thoughts, thanks!

**Comment by praveenjuge** [comment]
Yeah when I first started this, all my projects was coupled with alpine js so I started using it here too. But now it feels like it was a mistake to depend upon it because recently I had a need to use kutty in a react project and I couldn't. 

I think we can separate css and js, leaving the kutty tailwind plugin here, and make a separate project for js like kutty-alpine, kutty-react etc. 

Thanks for bringing this up! I'll let you know on further development here. 👍

**Comment by jerrynavi** [comment]
If you ever get started on this and need some help, I'll be happy to provide support however I can @praveenjuge
